frontend/src/components/orgs-list.ts

```diff
@@ -45,29 +44,34 @@ export class OrgsList extends LiteElement {
     }
 
     return html`
-    <btrix-dialog
-      label=${msg(str`Quotas for: ${this.currOrg.name}`)}
-      ?open=${!!this.currOrg}
-    @sl-request-close=${() => (this.currOrg = null)}
-  >
-  ${Object.entries(this.currOrg.quotas).map(([key, value]) => {
-    return html`
-    <sl-input
-    name=${key}
-    label=${msg("Max Concurrent Crawls")}
-    value=${value}
-    type="number"
-    @sl-input="${this.onUpdateQuota}"
-    ></sl-input>`;
-  })}
-  <div slot="footer" class="flex justify-end">
-    <sl-button 
-    size="small"
-    @click="${this.onSubmitQuotas}" 
-    variant="primary">${msg("Update Quotas")}
-    </sl-button>
-  </div>
-  </btrix-dialog>
+      <btrix-dialog
+        label=${msg(str`Quotas for: ${this.currOrg.name}`)}
+        ?open=${!!this.currOrg}
+        @sl-request-close=${() => (this.currOrg = null)}
+      >
+        ${Object.entries(this.currOrg.quotas).map(([key, value]) => {
+          const label =
+            key === "maxConcurrentCrawls"
+              ? "Max Concurrent Crawls"
```

Comment:
I think for localization to work correctly, it's better to just have:

```
...
              ? msg("Max Concurrent Crawls")
              : msg("Max Pages Per Crawl")
```

so that the correct strings get added to the string table.